api/v1beta1/cinder_types.go

```diff
@@ -49,6 +49,11 @@ type CinderSpec struct {
 	// TODO: -> implement needs work in mariadb-operator, right now only cinder
 	DatabaseUser string `json:"databaseUser"`
 
+	// +kubebuilder:validation:Required
+	// RabbitMQ instance name
+	// Needed to request a transportURL that is created and used in Cinder
+	RabbitMqClusterName string `json:"rabbitMqClusterName"`
```

Comment:
This is going to break `make cinder_deploy` and `make openstack_deploy` in `install_yamls` as they currently exist.  Assuming that having this field in the `CinderSpec` is the proper approach, how will we populated it for `install_yamls`?

Reply:
could we just have a default value set on this parameter to 'rabbitmq' to match the default name that we call it? Then it shouldn't break anything right

Reply:
@abays to follow up here I think we probably need to align our 'make rabbitmq_deploy' target with what we do in openstack. I pushed this to address the issue: https://github.com/openstack-k8s-operators/install_yamls/pull/41
Assuming we go this route I'd say setting the default here to 'rabbitmq' would be best right?

Reply:
-1: I agree, it's probably best to set the default to `rabbitmq`

Reply:
Ack, will do